client/ayon_zbrush/plugins/create/create_workfile.py

```diff
@@ -72,6 +73,6 @@ def create(self):
             product_name = self.get_product_name(
                 variant, task_entity, folder_entity, project_name, host_name
             )
-            current_instance["folderPath"] = folder_path
-            current_instance["task"] = task_entity
+            current_instance["task"] = task_entity["name"]
             current_instance["subset"] = product_name
```

Comment:
Shouldn't `subset` be `productName` nowadays?

Reply:
Let's take a reference from another host integration [here](https://github.com/ynput/ayon-core/blob/cef5787f29064012fa84d38c7ccb9f0bada8cda7/server_addon/houdini/client/ayon_houdini/plugins/create/create_workfile.py#L52-L93).

Task type should be redundant, but subset should be productName to align with latest refactoring?

Reply:
@BigRoy It's currently set as `subset` in the but, I will update it and do some testing to see if I have any issues with `productName`

Reply:
It seems to be functioning just fine with `productName` replacing `subset`